src/js/components/FormField/stories/RequiredLabel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, FormField, Form, Text, Button, Grommet } from 'grommet';
+
+const FormFieldLabel = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={
+        required ? (
+          <Box direction="row">
+            <Text>{label}</Text>
+            <Text color="red">*</Text>
```

Comment:
I think we would want to use grommet colors here, what about `status-error`? https://v2.grommet.io/color